src/App.tsx

```diff
@@ -1,172 +1,40 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './pages/HomePage/HomePage';
+import { PeopleTablePage } from './pages/PeopleTablePage/PeopleTablePage';
+import { PageNotFound } from './pages/PageNotFound/PageNotFound';
+import { NavBar } from './components/NavBar/NavBar';
 
 export const App = () => (
   <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
+    <NavBar />
 
     <main className="section">
       <div className="container">
```

Comment:
leave only `<Routes>`  tag in `App.tsx`

Create a `Layout` component and wrap all your routes in the `Layout` route.

```
<Routes>
       <Route path="/" element={<Layout/>}>
             ...
        </Route>
</Routes>
```
In Layout, you can use `Outlet` from `react-router-dom` to render your components inside